rasa/core/policies/keras_policy.py

```diff
@@ -70,6 +71,13 @@ def __init__(
 
         self.current_epoch = current_epoch
 
+        raise_warning(
+            "'KerasPolicy' is deprecated and will be removed in version "
+            "2.0. Use 'TEDPolicy' instead.",
+            category=FutureWarning,
+            docs=f"{DOCS_BASE_URL}/core/policies/",
```

Comment:
Import this constant as well
```suggestion
            docs=f"{DOCS_URL_POLICIES}",
```